discovery.rst

```diff
@@ -84,23 +84,52 @@ Read more about setting up Puli in their `official documentation`_.
 Common Errors
 -------------
 
+Could not find resource using any discovery strategy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you get an error saying "*Could not find resource using any discovery strategy.*" it means that all the
+discovery [#strategies]_ have failed. The cause of this is probably because you have not installed message factories
```

Comment:
cause and because don't work in the same sentence afaik.

Most likely, your project is missing the message factories and/or a PRS-7 implementation.